gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/resource/integration/IntegrationsResource_CreateTest.java

```diff
@@ -0,0 +1,111 @@
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.doReturn;
+
+import io.gravitee.common.http.HttpStatusCode;
+import io.gravitee.repository.exceptions.TechnicalException;
+import io.gravitee.rest.api.management.v2.rest.model.CreateIntegration;
+import io.gravitee.rest.api.management.v2.rest.model.Integration;
+import io.gravitee.rest.api.management.v2.rest.resource.AbstractResourceTest;
+import io.gravitee.rest.api.model.EnvironmentEntity;
+import io.gravitee.rest.api.service.common.GraviteeContext;
+import io.gravitee.rest.api.service.common.UuidString;
+import jakarta.ws.rs.client.Entity;
+import jakarta.ws.rs.client.WebTarget;
+import jakarta.ws.rs.core.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class IntegrationsResource_CreateTest extends AbstractResourceTest {
+
+    private static final String ENVIRONMENT = "my-env";
+    private static final String INTEGRATION_NAME = "test-name";
+    private static final String INTEGRATION_DESCRIPTION = "integration-description";
+    private static final String INTEGRATION_PROVIDER = "test-provider";
+    private static final String INTEGRATION_ID = "generated-id";
+
+    WebTarget target;
+
+    @Override
+    protected String contextPath() {
+        return "/environments/" + ENVIRONMENT + "/integrations";
+    }
+
+    @BeforeEach
+    public void init() throws TechnicalException {
+        target = rootTarget();
+
+        EnvironmentEntity environmentEntity = EnvironmentEntity.builder().id(ENVIRONMENT).organizationId(ORGANIZATION).build();
+        doReturn(environmentEntity).when(environmentService).findById(ENVIRONMENT);
+        doReturn(environmentEntity).when(environmentService).findByOrgAndIdOrHrid(ORGANIZATION, ENVIRONMENT);
+
+        GraviteeContext.setCurrentEnvironment(ENVIRONMENT);
+        GraviteeContext.setCurrentOrganization(ORGANIZATION);
+
+        UuidString.overrideGenerator(() -> INTEGRATION_ID);
+    }
+
+    @Test
+    public void should_create_integration() {
+        //Given
+        CreateIntegration createIntegration = CreateIntegration
+            .builder()
+            .name(INTEGRATION_NAME)
+            .description(INTEGRATION_DESCRIPTION)
+            .provider(INTEGRATION_PROVIDER)
+            .build();
+
+        //When
+        Response response = target.request().post(Entity.json(createIntegration));
+
+        //Then
+        assertThat(response.getStatus()).isEqualTo(HttpStatusCode.CREATED_201);
+
+        var integrationResponse = response.readEntity(Integration.class);
+        assertThat(integrationResponse).isNotNull();
+        assertThat(integrationResponse)
+            .extracting(Integration::getId, Integration::getName, Integration::getDescription, Integration::getProvider)
+            .containsExactly(INTEGRATION_ID, INTEGRATION_NAME, INTEGRATION_DESCRIPTION, INTEGRATION_PROVIDER);
+    }
```

Comment:
You can have a single instruction to handle the assertion. (https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/resource/group/GroupsResourceTest.java#L119)